gigascience/gigadb-website
# Update adminDataset form with custom field components

# Pull request for issue: #1558

This is a pull request for the following functionalities:

* Use custom components that have built-in a11y features in  http://gigadb.gigasciencejournal.com:9170/adminDataset/create // http://gigadb.gigasciencejournal.com:9170/adminDataset/update/id/5 form, for the ones where that doesn't apply, add a11y features wherever they are missing

## How to test?

* Run WAVE audit on above pages, there should be no errors reported, except for a duplicate checkbox that is probably due to the mock DB
* Run playwright tests `cd playwright; npm i; npm test`, all running tests should pass

## How have functionalities been implemented?

Used custom field components wherever suitable. For cases where components didn't fit, a11y features (e.g. role="alert" for errors, "required" attribute, etc) were added

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
